psydac/linalg/tests/test_solvers.py

```diff
@@ -102,31 +102,53 @@ def test_solver_tridiagonal(n, p, dtype, solver, verbose=False):
         print()
 
     #Create the solvers
-    solv  = inverse(A, solver, tol=1e-13, verbose=True)
+    solv  = inverse(A, solver, tol=tol, verbose=False, recycle=True)
```

Comment:
If we pass both `x0` and `recycle=True`, will the vector `x0` be modified?

Reply:
Yes. If we don't pass `x0`, we immediately set `x0 = codomain.zeros()`, so there's no possible case where `x0 = None` might change the way any internal functions work.